docs/1.getting-started/5.seo-meta.md

```diff
@@ -172,9 +172,9 @@ It's recommended to use getters (`() => value`) over computed (`computed(() => v
 
 You can use the `titleTemplate` option to provide a dynamic template for customizing the title of your site. For example, you could add the name of your site to the title of every page.
 
-The `titleTemplate` can either be a string, where `%s` is replaced with the title, or a function.
+The `titleTemplate` can either be a string, where `%s` is replaced with the title or a function.
```

Comment:
This isn't correct. The comma finishes the `where s is replaced with the title` clause. The template can either be a string or a function.

```suggestion
The `titleTemplate` can either be a string, where `%s` is replaced with the title, or a function.
```